monorepo/monorepo-ws/apps/articleapp/src/Schema/article/index.test.ts

```diff
@@ -0,0 +1,54 @@
+import { gql } from "apollo-server";
+import { server as testServer } from '../../test-utils'
+
+const queryArticle = gql`
+    query{
+        articles{
+            id,
+            title,
+            authorId
+        }
+    }
+`
+const getArticles = gql`
+    query {
+        getAllArticles {
+            id,
+            title,
+            authorId
+        }
+    }
+`
+
+describe('article resolver', () => {
+    it('should return articles', async () =>{
+        const result = await testServer.executeOperation({
+            query: queryArticle
+        }) 
+        expect(result.data).toMatchSnapshot({articles: articleMatcher})        
+    }),
+    it('shoult return articles from getArticles', async () => {
+        const result = await testServer.executeOperation({
+            query: getArticles
+        })
+        expect(result.data).toMatchSnapshot({getAllArticles: articleMatcher})
+    })
+})
+
+const articleMatcher = [
```

Comment:
🧹 tohle je to same jeste  myslim ze to nema  vyznam tady mit jakejkoliv matcher, protoze kdybys rozsiril ten objekt  tak to musis tady rozsirovat tez

Reply:
odstraněno

Reply:
to uplne nevim, jsi to jen posunul, ale ty nemusis tomu toMatchSnapshot nic a pak se to chova automaticky 
<img width="573" alt="image" src="https://user-images.githubusercontent.com/14004390/183588990-5d03bce6-a13d-4934-871c-62094eb28d67.png">